public/components/datasources/components/manage/associated_objects_tab.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiHorizontalRule,
+  EuiButton,
+  EuiSpacer,
+} from '@elastic/eui';
+import { AccelerationsRecommendationCallout } from './accelerations_recommendation_callout';
+
+interface AssociatedObject {
+  id: string;
+  name: string;
+  database: string;
+  type: string;
+  createdByIntegration: string;
+  accelerations: string;
+}
```

Comment:
Could be exported out of [`dashboards-observability/common/types/data_connections.ts`](https://github.com/opensearch-project/dashboards-observability/blob/main/common/types/data_connections.ts), since other files could use the same object type.

Reply:
done